[2023-10-06T07:33:23.133+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_to_bigquery_dag.wait_for_file manual__2023-10-06T07:33:20.881485+00:00 [queued]>
[2023-10-06T07:33:23.142+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_to_bigquery_dag.wait_for_file manual__2023-10-06T07:33:20.881485+00:00 [queued]>
[2023-10-06T07:33:23.143+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-06T07:33:23.155+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonSensor): wait_for_file> on 2023-10-06 07:33:20.881485+00:00
[2023-10-06T07:33:23.194+0000] {standard_task_runner.py:57} INFO - Started process 13013 to run task
[2023-10-06T07:33:23.205+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'load_to_bigquery_dag', 'wait_for_file', 'manual__2023-10-06T07:33:20.881485+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/load_to_bigquery_dag.py', '--cfg-path', '/tmp/tmpz35dqtbf']
[2023-10-06T07:33:23.212+0000] {standard_task_runner.py:85} INFO - Job 58: Subtask wait_for_file
[2023-10-06T07:33:23.317+0000] {task_command.py:415} INFO - Running <TaskInstance: load_to_bigquery_dag.wait_for_file manual__2023-10-06T07:33:20.881485+00:00 [running]> on host bd744fef73d7
[2023-10-06T07:33:23.455+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='load_to_bigquery_dag' AIRFLOW_CTX_TASK_ID='wait_for_file' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T07:33:20.881485+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-06T07:33:20.881485+00:00'
[2023-10-06T07:33:23.458+0000] {python.py:72} INFO - Poking callable: <function check_files_exist at 0xffff9099b310>
[2023-10-06T07:33:23.461+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:28.484+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:33.519+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:38.551+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:43.559+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:48.568+0000] {logging_mixin.py:151} INFO - Not all files exist. Retrying in 5 seconds...
[2023-10-06T07:33:53.579+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-06T07:33:53.611+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=load_to_bigquery_dag, task_id=wait_for_file, execution_date=20231006T073320, start_date=20231006T073323, end_date=20231006T073353
[2023-10-06T07:33:53.696+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
